internal/testrunner/coverageoutput.go

```diff
@@ -57,48 +64,53 @@ type coberturaCoverage struct {
 	BranchesCovered int64               `xml:"branches-covered,attr"`
 	BranchesValid   int64               `xml:"branches-valid,attr"`
 	Complexity      float32             `xml:"complexity,attr"`
-	Sources         []*coberturaSource  `xml:"sources>source"`
-	Packages        []*coberturaPackage `xml:"packages>package"`
+	Sources         []*CoberturaSource  `xml:"sources>source"`
+	Packages        []*CoberturaPackage `xml:"packages>package"`
 }
 
-type coberturaSource struct {
+// CoberturaSource represents a base path to the covered source code.
+type CoberturaSource struct {
 	Path string `xml:",chardata"`
 }
 
-type coberturaPackage struct {
+// CoberturaPackage represents a package in a Cobertura XML report.
+type CoberturaPackage struct {
 	Name       string            `xml:"name,attr"`
 	LineRate   float32           `xml:"line-rate,attr"`
 	BranchRate float32           `xml:"branch-rate,attr"`
 	Complexity float32           `xml:"complexity,attr"`
-	Classes    []*coberturaClass `xml:"classes>class"`
+	Classes    []*CoberturaClass `xml:"classes>class"`
 }
 
-type coberturaClass struct {
+// CoberturaClass represents a class in a Cobertura XML report.
+type CoberturaClass struct {
 	Name       string             `xml:"name,attr"`
 	Filename   string             `xml:"filename,attr"`
 	LineRate   float32            `xml:"line-rate,attr"`
 	BranchRate float32            `xml:"branch-rate,attr"`
 	Complexity float32            `xml:"complexity,attr"`
-	Methods    []*coberturaMethod `xml:"methods>method"`
+	Methods    []*CoberturaMethod `xml:"methods>method"`
```

Comment:
You will have to update the docs with new mapping information: https://github.com/elastic/integrations/blob/master/docs/testing_and_validation.md#review-test-coverage